.github/workflows/tag.yml

```diff
@@ -0,0 +1,111 @@
+        elif [ $IS_MINOR == 'true' ]
+        then
+          echo "::set-env name=BV_PART::minor"
+        else
+          echo "::set-env name=BV_PART::patch"
+        fi
+    - name: Checkout repository
+      uses: actions/checkout@v1.2.0
+    - name: Set up git user
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+    - name: Checkout master
+      run: git checkout master
+    - name: Squash and merge PR#${{ github.event.pull_request.number }} to master
+      if: github.event.action == 'labeled'
+      run: |
+        git merge --squash "origin/${GITHUB_HEAD_REF}"
+        git commit -m "${PR_TITLE} (#${PR_NUMBER})"
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install bumpversion
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install bumpversion
+    - name: Run bumpversion ${{ env['BV_PART'] }}
+      run: |
+        OLD_TAG=$(git describe --tags --abbrev=0)
+        echo "::set-env name=OLD_TAG::${OLD_TAG}"
+        bumpversion $BV_PART --message "Bump version: {current_version} → {new_version}
+
+        Triggered by #${PR_NUMBER} via GitHub Actions."
+        NEW_TAG=$(git describe --tags --abbrev=0)
+        echo "::set-env name=NEW_TAG::${NEW_TAG}"
+        git tag -n99 -l $NEW_TAG
+
+        CHANGES=$(git log --pretty=format:'%s' $OLD_TAG..HEAD -i -E --grep='^([a-z]*?):')
+        CHANGES_NEWLINE="$(echo "${CHANGES}" | sed -e 's/^/  - /')"
+        SANITIZED_CHANGES=$(echo "${CHANGES}" | sed -e 's/^/<li>/' -e 's|$|</li>|' -e 's/(#[0-9]\+)//' )
+        echo "::set-env name=CHANGES::${SANITIZED_CHANGES//$'\n'/}"
+        NUM_CHANGES=$(echo -n "$CHANGES" | grep -c '^')
+        echo "::set-env name=NUM_CHANGES::${NUM_CHANGES}"
+        git tag $NEW_TAG $NEW_TAG^{} -f -m "$(printf "This is a $BV_PART release from $OLD_TAG → $NEW_TAG.\n\nChanges:\n$CHANGES_NEWLINE")"
+        git tag -n99 -l $NEW_TAG
+    - name: Comment on issue
+      if: >-
+        github.event.action == 'labeled'
+      uses: actions/github-script@0.3.0
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          github.issues.createComment({...context.issue, body: "I've queued this up. When it gets merged, I'll create a ${{ env['BV_PART'] }} release from ${{ env['OLD_TAG'] }} → ${{ env['NEW_TAG'] }} which includes the following ${{ env['NUM_CHANGES'] }} change(s) [including this PR]:<br />${{ env['CHANGES'] }}<br />If you make any more changes, you probably want to re-trigger me again by removing the `bumpversion/${{ env['BV_PART'] }}` label and then adding it back again."})
+    - name: Push changes
+      if: >-
+        github.event.action == 'closed' && github.event.pull_request.merged
+      uses: ad-m/github-push-action@v0.5.0
+      with:
+        github_token: ${{ secrets.GITHUB_PAT }}
```

Comment:
@andrzejnovak before this can be ready for review you need to [make and add a GitHub personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) as a secret named `GITHUB_PAT`.

Reply:
are specific scopes needed?

Reply:
Good question. I'm not sure, so we'll have to ask @kratsg, but I think only `repo` is needed.

Reply:
Let's try

Reply:
Hm. There is an error. 

```
fatal: tag 'v0.0.26' already exists
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.5/x64/bin/bumpversion", line 8, in <module>
    sys.exit(main())
  File "/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/bumpversion/__init__.py", line 1004, in main
    vcs.tag(tag_name)
  File "/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/bumpversion/__init__.py", line 150, in tag
    subprocess.check_output(["git", "tag", name])
  File "/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'tag', 'v0.0.26']' returned non-zero exit status 128.
```

I'll need to debug this later.

Reply:
Oh. @andrzejnovak [v`0.0.26` is on PyPI](https://pypi.org/project/mplhep/0.0.26/) already. Did you publish this earlier not through GitHub? :?

Reply:
Ah yes. https://github.com/scikit-hep/mplhep/commit/5a3b9cd3f919dcde84c69be9e16eb14bbad221e4 Just not on master, which is why this workflow is confused. Can you please merge 5a3b9cd3f919dcde84c69be9e16eb14bbad221e4 to master?